boards/arm/mps2/Kconfig.mps2

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2017 Linaro Limited
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_MPS2_AN385
+	bool
+	select SOC_MPS2_AN385
+	select QEMU_TARGET
+	select HAS_COVERAGE_SUPPORT
```

Comment:
should go to `Kconfig` file.
Same comment applies to below lines.

Secondly, all SoCs and variants has:
```
	select QEMU_TARGET
	select HAS_COVERAGE_SUPPORT
```

so why not put this for the highlevel board name, as:
```
config BOARD_MPS2
	select QEMU_TARGET
	select HAS_COVERAGE_SUPPORT
```
and remove the duplicated entries for `_AN385`, `_AN521_CPU0`, `_AN521_CPU0_NS`, , `_AN521_CPU1`.

Reply:
Fixed